content/articles/2018-07-broom-0-5-0.Rmarkdown

```diff
@@ -0,0 +1,127 @@
+---
+title: broom 0.5.0
+author: Alex Hayes
+date: "2018-07-18"
+slug: broom-0-5-0
+description: > 
+  broom 0.5.0 is on CRAN.
+categories:
+  - package
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+options(tibble.print_min = 5, tibble.print_max = 5)
+```
+
+We are delighted to announced that [broom](https://broom.tidyverse.org/) 0.5.0 is now available on CRAN. broom 0.5.0 is a major new release featuring new changes that affect both users and developers. See the [News]( https://broom.tidyverse.org/news/) for a detailed list of changes.
+
+## Tibble output
+
+All tidiers should now return `tibble`s rather than `data.frame`s. This allows broom to take advantage of the nice tibble print method and the more consistent behavior of tibbles:
+
+```{r}
+library(broom)
+
+fit <- lm(mpg ~ ., mtcars)
+tidy(fit)
+```
+
+
+These changes will mostly likely affect you when:
+
+- subset with `[`, which always returns a tibble
+- set rownames on a tibble, which is deprecated
+- use augment methods on models with matrix covariates specified in a formula, which will error
+
+Augment will error with matrix covariates because tibbles are more strict about their contents than data frames. Data frames allow users to specify columns in a matrix, like so:
+
+```{r}
+y <- 1:3
+x <- matrix(1:9, nrow = 3)
+df <- data.frame(x, y)
+```
+
+Tibbles do not:
+
+```{r, error = TRUE}
+library(tibble)
+
+tibble(x, y)
+```
+
+Modelling functions will occassional create a data frame like this, but since the model frame can't be coerced a tibble method, `augment` will fail:
+
+```{r, error = TRUE}
+fit <- lm(y ~ x, df)
+augment(fit)
+```
+
+In some cases, explicitly passing the original dataset via the `data` argument can resolve this:
+
+```{r}
+augment(fit, data = df)
+```
+
+In any case, support for matrix-columns is on the way in dplyr and in a release cycle or two this won't be an issue.
+
+A number of deprecated tidiers (or tidiers that will be deprecated in the next release) have not been updated to use tibble output. Most notably, this includes tidiers for mixed models. More on this below.
+
+## New infrastructure for testing and documentation
+
+We are heavily invested in making it easier to contribute to broom, and also in making broom behavior more standardized and consistent. To this end, we've written new testing and documentation infrastructure that automates much of these processes.
+
+In broom 0.5.0 these tests mostly ensure tibble output along with one or two other consistency items. However, stricter versions of these tests are under development for future releases.
+
+We have also migrated to a new template based documentation strategy. Now repeated documentation material lives in `roxygen2` templates and can easily added to a new tidy method. For an example of how this works, see [`R/aer-tidiers.R`](https://github.com/alexpghayes/broom/blob/master/R/aer-tidiers.R).
```

Comment:
Hyphenate template-based (compound modifier 😉).

Reply:
* Either comma after that opening "Now" or flip to "Repeated documentation material _now_ lives in…" (emphasis added just to point out where the change is)
* Missing "be" before added (can easily _be_ added)